students/sungho-moon/westagram/urls.py

```diff
@@ -14,3 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+           path('westagram', include('user.urls'))
```

Comment:
user와 관련된 app으로 연결 시켜주므로 아래와 같이 지정하는 것이 좋습니다.
```suggestion
           path('user', include('user.urls'))
```

Reply:
수정완료했습니다